res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -10,25 +10,12 @@ var TrackEndWarning = true;
 // when SHIFT enabled and scratching with the jog wheel
 var iCutEnabled = true;
 
-// fastSeekEnabled: enable fast seek with Jog Wheel with Wheel Off and Shift ON 
-// Shift can be locked or not
-var fastSeekEnabled = true;
-
 //activate PFL of deck on track load
 var smartPFL = true;
 
 // use beatlooproll instead of beatloop
 var beatlooprollActivate = false;
```

Comment:
I don't think those behaviours are the same, this option makes it so that all the loops set with the auto loop buttons are rolling. Also, the Shift behaviour isn't to toggle loopRoll, but rather to allow for the toggling of fractional beat loops (mapping 4 buttons to 8 loop sizes).

I understand your suggestion but I'm not quite sure if I want all of my <= 1 beat loops to be rolling. Will consider it, especially if others want the shifted behaviour you describe.

For now, I will remove the option since I can't imagine anyone wanting all of their loops to be rolling, especially without a way to toggle. Will be revisiting this when #1187 is merged.